fdbserver/VersionedBTree.actor.cpp

```diff
@@ -4877,11 +5088,13 @@ class VersionedBTree {
 			}
 
 			auto& metrics = g_redwoodMetrics.level(btPage->height);
```

Comment:
```suggestion
			auto& metrics = g_redwoodMetrics.level(height);
```
This line isn't changed by the PR but should also just use `height` for clarity.